.circleci/config.yml

```diff
@@ -901,14 +901,14 @@ workflows:
           requires:
             - cleanup-azure-resources
       - acceptance-kind-1-21
-#  update-helm-charts-index: <-- Disable until we can figure out a release workflow. We currently don't want it to trigger on a tag because the tag is pushed by the eco-releases pipeline.
-#    jobs:
-#      - helm-chart-pipeline-approval:
-#          type: approval
-#      - update-helm-charts-index:
-#          requires:
-#            - helm-chart-pipeline-approval
-#          context: helm-charts-trigger-consul
+  update-helm-charts-index: # <-- Disable until we can figure out a release workflow. We currently don't want it to trigger on a tag because the tag is pushed by the eco-releases pipeline.
+    jobs:
+      - helm-chart-pipeline-approval:
+          type: approval
+      - update-helm-charts-index:
+          requires:
+            - helm-chart-pipeline-approval
+          context: helm-charts-trigger-consul
 #          filters:
```

Comment:
Only comment in the jobs themselves, not the filters. This way, you have control over triggering the pipeline with the branch you create. You'll need to approve the approval step to kick off the helm charts pipeline.